social-messenger-ts/src/hooks/useUpdateAppHeightOnResize.ts

```diff
@@ -0,0 +1,25 @@
+import { useEffect } from 'react';
+
+/**
+ * Internal, updates CSS variable with the current window height
+ * upon window resize event.
+ */
+export const useUpdateAppHeightOnResize = () => {
+  useEffect(() => {
+    /*
+     * Get the actual rendered window height to set the container size properly.
+     * In some browsers (like Safari) the nav bar can override the app.
+     */
+    const setAppHeight = () => {
+      const doc = document.documentElement;
+      doc.style.setProperty('--app-height', `${window.innerHeight}px`);
```

Comment:
Idea for future, we can probably use one of [the relative units for that](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#numbers_lengths_and_percentages).